Tensile/TensileCreateLibrary.py

```diff
@@ -751,7 +764,7 @@ def success(kernel):
         (err, src) = ko.getSourceFileString()
         kernelSourceFile.write(src)
         if err:
-            print("*** warning: invalid kernel#%u" % kernelName)
+            printWarning("*** warning: invalid kernel#%u" % kernelName)
```

Comment:
Is it redundant to have warning in the string?

Reply:
Yes it is, will remove, good catch.